packages/office-ui-fabric-react/src/components/Rating/__snapshots__/Rating.test.tsx.snap

```diff
@@ -24,6 +24,8 @@ exports[`Rating Renders Rating correctly 1`] = `
         height: 32px;
       }
   id="Rating0"
+  max={5}
```

Comment:
Max and min do not look right..

Reply:
i'll get a separate pr going to fix the getNativeProps.